src/ralph/export_to_ng/resources.py

```diff
@@ -216,6 +217,74 @@ def get_queryset(self):
         )
 
 
+class NetworkResource(resources.ModelResource):
+    kind = fields.Field('kind', column_name='kind')
+    # skipped until scan is implemented
+    # network_environment = fields.Field(
+    #     'network_environment', column_name='network_environment',
+    # )
+
+    def dehydrate_kind(self, network):
+        return network.kind_id or ''
+
+    def dehydrate_network_environment(self, network):
+        return network.environment_id or ''
+
+    class Meta:
+        fields = (
+            'id',
+            'name',
+            'created',
+            'modified',
+            'address',
+            #TODO:: blank charfield?
+            'gateway',
+            'gateway_as_int',
+            'reserved',
+            'reserved_top_margin',
+            #TODO:: blank charfield
+            'remarks',
+            'vlan',
+            'min_ip',
+            'max_ip',
+            'ignore_addresses',
+            'dhcp_broadcast',
+            #TODO:: blank textfield
+            'dhcp_config',
+            'kind',
+            # 'network_environment',
+        )
+        model = models_network.Network
+
+
+class IPAddressResource(resources.ModelResource):
+    asset = fields.Field('asset', column_name='asset')
+    def dehydrate_asset(self, ip_address):
+        return ip_address.device_id or ''
```

Comment:
are you sure that by exporting device_id we'll be able to match it to asset in ng (which is mapped probably to asset_id in mapping table)?